go.mod

```diff
@@ -269,6 +268,7 @@ replace (
 replace (
 	// Use cosmos keyring
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
+	github.com/cosmos/iavl => github.com/cosmos/iavl v1.1.2
```

Comment:
getting mismatch after bump iavl 
https://github.com/cosmos/iavl/compare/v1.1.2...v1.2.0

Reply:
can you add a `// FIXME`, we can fix in another PR